spec/models/order_spec.rb

```diff
@@ -17,35 +16,87 @@
   end
 
   describe '#fulfilled?' do
-    # These are gross, I want to use one-liner syntax
     subject { build(order_type) }
 
+    it { is_expected.to_not be_fulfilled }
+
     context 'when fulfilled_at is set' do
-      it 'it is fulfilled' do
+      it 'is fulfilled' do
         subject.fulfilled_at = Time.now
 
         expect(subject).to be_fulfilled
       end
     end
+  end
+
+  describe '#canceled?' do
+    subject { build(order_type) }
+
+    it { is_expected.to_not be_canceled }
+
+    context 'when status is set to canceled' do
+      it 'is canceled' do
+        subject.status = 'canceled'
```

Comment:
I like to put things that are just about the context in a `before` block inside of the context.

So `before { subject.status = 'canceled' }`.